packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -244,6 +258,7 @@ export class ParticleGenerator {
    * @internal
    */
   _update(elapsedTime: number): void {
+    const lastAlive = this.isAlive;
```

Comment:
`isAlive` contain play state maybe use `this._firstActiveElement !== this._firstFreeElement`